types/xelib/index.d.ts

```diff
@@ -0,0 +1,1629 @@
+     */
+    ReleaseNodes(id: Handle): void;
+    /**
+     * Finds other handles to a particular interface.
+     */
+    GetDuplicateHandles(id: Handle): Handle[];
+
+    /**
+     * Message functions
+     * @see {@link https://z-edit.github.io#/docs?t=Development%2FAPIs%2Fxelib%2FMessages}
+     */
+    /**
+     * Gets any messages that have been added to XEditLib's internal log
+     * since the last time this function was called.
+     */
+    GetMessages(): string;
+    /**
+     * Clears all messages from the XEditLib's internal log.
+     */
+    ClearMessages(): void;
+    /**
+     * Returns a message corresponding to the last exception that occurred.
+     * Returns an empty string if no exception has occurred since the last time this function was called.
+     */
+    GetExceptionMessage(): string;
+
+    /**
+     * Setup functions
+     * @see {@link https://z-edit.github.io#/docs?t=Development%2FAPIs%2Fxelib%2FSetup}
+     */
+    /**
+     * @see LoaderState
+     */
+    loaderStates: typeof LoaderState;
+    /**
+     * @see GameMode
+     */
+    gameModes: typeof GameMode;
+    /**
+     * Array of game objects corresponding to each supported game mode.
+     */
+    games: readonly Game[];
+    /**
+     * Retrieves the path to the game corresponding to gameMode from the registry.
+     * Returns an empty string if the game path cannot be found.
+     * @see GameMode
+     */
+    GetGamePath(gameMode: GameMode): string;
+    /**
+     * Sets the game path to be used when loading plugin and resource files to gamePath.
+     */
+    SetGamePath(gamePath: string): void;
+    /**
+     * Retrieves the the language used for gameMode from the game INI file.
+     * Returns an empty string if the game INI file cannot be found.
+     * @see GameMode
+     */
+    GetGameLanguage(gameMode: GameMode): string;
+    /**
+     * Sets the language to be used when loading string files to language.
+     */
+    SetLanguage(language: string): void;
+    /**
+     * Sets the game mode to gameMode.
+     * @see GameMode
+     */
+    SetGameMode(gameMode: GameMode): void;
+    /**
+     * Returns the user's load order determined from loadorder.txt, plugins.txt, or plugin dates depending on the game and available files.
+     * The load order is returned as a list of filenames separated by \r\n.
+     */
+    GetLoadOrder(): string;
+    /**
+     * Returns the user's active plugins determined from plugins.txt.
+     * Active plugins are returned as a list of filenames separated by \r\n.
+     */
+    GetActivePlugins(): string;
+    /**
+     * Loads plugin files in loadOrder.
+     * If smartLoad is set to true, master files required by files in loadOrder will be automatically loaded as necessary.
+     * Plugin loading is performed in a background thread, use GetLoaderStatus to track the loader and determine when it is done.
+     * @see GetLoaderStatus
+     */
+    LoadPlugins(loadOrder: string, smartLoad: boolean): void;
```

Comment:
```suggestion
    LoadPlugins(loadOrder: string, smartLoad?: boolean): void;
```
https://github.com/z-edit/xelib/blob/72c8dd97956098947b49fbdeadd0e32fd8ca0d38/src/js/setup.js#L80